lib/galaxy/tools/deps/containers.py

```diff
@@ -303,6 +306,9 @@ def add_var(name, value):
         else:
             defaults = "$galaxy_root:ro,$tool_directory:ro,$working_directory:rw,$default_file_path:rw"
 
+        if self.app_info.library_import_dir:
+            defaults = defaults + ",$library_import_dir:ro"
```

Comment:
Could be also:

`defaults += ",$library_import_dir:ro"`

But I guess it depends on how much you want your code to be obvious :)